docker/jupyter_notebook_config.py

```diff
@@ -0,0 +1,17 @@
+# pulled from https://hub.docker.com/r/ansingh7115/leonardo-notebooks/
+# and added base_url configs for local proxy testing
+
+from jupyter_core.paths import jupyter_data_dir
+import subprocess
+import os
+import errno
+import stat
+
+c = get_config()
+c.NotebookApp.ip = '*'
+c.NotebookApp.port = 8000
+c.NotebookApp.open_browser = False
+c.NotebookApp.base_url = '/notebooks/test/'
```

Comment:
I haven't seen notebooks-land in a while, but IIRC these options let you set the notebooks landing page to `host.com/things_you_specify/tree` rather than just `host.com/tree`. Why does the proxy care about this, beyond having to know what it is?